packages/ERTP/test/unitTests/test-mintObj.js

```diff
@@ -49,7 +52,7 @@ test('mint.mintPayment strSetMathHelpers', t => {
 
 test('mint.mintPayment setMathHelpers', t => {
   t.plan(3);
-  const { mint, issuer, amountMath } = makeIssuerKit('items', 'set');
+  const { mint, issuer, amountMath } = makeIssuerKit('items', SET);
```

Comment:
e.g., this would seem more readable:
```suggestion
  const { mint, issuer, amountMath } = makeIssuerKit('items', MATH_SET);
```

Reply:
Or alternatively, `MathKind.SET`